e2e/step_definitions/programCatalog.js

```diff
@@ -82,4 +86,31 @@ defineSupportCode(({
         return expect(ProgramCatalog.isCorrectCourseLevelTextVisible(level)).to.eventually.be.true;
     });
 
+    Then(/^the "([^"].*)" card's details expander is clicked$/, text => {
+        return ProgramCatalog.clickOnTheExpander(text);
+    });
+
+    Then(/^the "([^"].*)" card's short description should be visible$/, text => {
```

Comment:
Please write this function the way so that it would check the hidden condition as well.